x/ccv/provider/keeper/grpc_query.go

```diff
@@ -333,17 +335,59 @@ func (k Keeper) QueryConsumerValidators(goCtx context.Context, req *types.QueryC
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	if _, found := k.GetConsumerClientId(ctx, consumerId); !found {
-		// chain has to have started; consumer client id is set for a chain during the chain's spawn time
-		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("no started consumer chain: %s", consumerId))
+	// check that the consumer is initialized
+	initParams, err := k.GetConsumerInitializationParameters(ctx, consumerId)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, errorsmod.Wrap(types.ErrUnknownConsumerId, consumerId).Error())
```

Comment:
Validators can opt-in on a consumer that is just created without the init params being set. This means the query should return something even if there are no init params.